content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,576 @@
+cluster.
+
+![How the Target Allocator's service discovery feature works](target-allocator-promcr.png)
+
+Even though Prometheus is not required to be installed in your Kubernetes 
+cluster to use the Target Allocator for Prometheus CR discovery, the TA does 
+require that the `ServiceMonitor` and `PodMonitor` be installed. These CRs are 
+bundled with Prometheus Operator; however, they can be installed standalone as 
+well. The easiest way to do this is to grab a copy of the individual 
+[PodMonitor YAML](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/charts/crds/crds/crd-podmonitors.yaml) and 
+[ServiceMonitor YAML](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/charts/crds/crds/crd-servicemonitors.yaml) custom resource 
+definitions (CRDs).
+
+OTel supports the `PodMonitor` and `ServiceMonitor` Prometheus resources because 
+these are widely-used in Kubernetes infrastructure monitoring. As a result, the 
+OTel Operator developers wanted to make it easy to add them to the OTel 
+ecosystem.
+
+Note that the `PodMonitor` and `ServiceMonitor` are not useful for 
+cluster-wide metrics collection, such as for Kubelet metrics collection. In that 
+case, you still have to rely on Prometheus scrape configs in the Collector’s 
+[Prometheus Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md).
+
+#### Configuration
+The following is the YAML config for the OTel Collector CR. Note that this 
+Collector is running in a namespace called `opentelemetry`, but it can run 
+in whatever namespace you like. 
+
+The main components are:
+* **mode:** This is one of four [OTel Collector deployment modes supported by the Operator](https://github.com/open-telemetry/opentelemetry-operator?tab=readme-ov-file#deployment-modes): 
+Sidecar, Deployment, StatefulSet and DaemonSet. 
+* **targetallocator:** This is where you configure the Target Allocator. Note 
+that the [Target Allocator only works for the Deployment, DaemonSet, and StatefulSet modes](https://www.youtube.com/watch?v=Uwq4EPaMJFM). 
+* **config:** This is where the OTel Collector config resides.
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otelcol
+  namespace: opentelemetry
+spec:
+  mode: statefulset
+  targetAllocator:
+    enabled: true
+    serviceAccount: opentelemetry-targetallocator-sa
+    prometheusCR:
+      enabled: true
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+      prometheus:
+        config:
+          scrape_configs:
+          - job_name: 'otel-collector'
+            scrape_interval: 30s
+            static_configs:
+            - targets: [ '0.0.0.0:8888' ]
+        target_allocator:
+          endpoint: http://otelcol-targetallocator
+          interval: 30s
+          collector_id: "${POD_NAME}"
```

Comment:
[note: we actually do this automatically for users now!](https://github.com/open-telemetry/opentelemetry-operator/blob/main/pkg/featuregate/featuregate.go#L58-L65)